packages/backend/src/payment-method/ilp/connector/core/middleware/error-handler.ts

```diff
@@ -11,6 +11,12 @@ export function createIncomingErrorHandlerMiddleware(
   serverAddress: string
 ): ILPMiddleware {
   return async (ctx: ILPContext, next: () => Promise<void>): Promise<void> => {
+    const stopTimer = ctx.services.telemetry.startTimer(
```

Comment:
This middleware doesn't do much other than wrap all the other middlewares, so the timing of this will usually just equal the whole packet lifecycle, so as long as we are tracking that (I believe we do) then we don't necessarily need this one here